src/test/ui/rustdoc/doc-test-attr.rs

```diff
@@ -0,0 +1,16 @@
+#![crate_type = "lib"]
+#![deny(unused_attributes)]
+
+#![doc(test)]
+//~^ ERROR
```

Comment:
Why have this in both rustdoc and rustc? Doesn't it make sense to only have the rustc test?

Reply:
We need to ensure that the warning is emitted from both the compiler and rustdoc.

Reply:
Hmm, ok, I guess I buy that since rustdoc silences some lints. Can you add a comment to keep the two files in sync?

Reply:
Normally all tests in `src/test/ui/rustdoc` have to be present in `rustdoc-ui` as well.

Reply:
That was not apparent to me. Can you add a note to the readme in ui/rustdoc?

Reply:
I'll extend the existing README content. :)